cmake/Deps_Unix.cmake

```diff
@@ -79,30 +79,25 @@ endif ()
 if (WITH_ZIM)
     if (APPLE)
         # ICU from homebrew is "key-only", we need to manually prioritize it -> see `brew info icu4c`
-        set(ENV{PKG_CONFIG_PATH} "$ENV{PKG_CONFIG_PATH}:/usr/local/opt/icu4c@76/lib/pkgconfig:/opt/homebrew/opt/icu4c@76/lib/pkgconfig:/usr/local/opt/icu4c/lib/pkgconfig:/opt/homebrew/opt/icu4c/lib/pkgconfig")
+        # And we needs to find the correct one if multiple versions co exists.
+        set(ENV{PATH} "$ENV{PATH}:/usr/local/bin/:/opt/homebrew/bin") # add brew command into PATH
+        execute_process(
+                COMMAND sh -c [=[brew --prefix $(brew deps libzim | grep icu4c)]=]
```

Comment:
This is the proper way to find icu4c, because there can be multiple versions of it.

There are 3 versions as of now 74,75,76

See other versions -> https://formulae.brew.sh/formula/icu4c@76